proto/translate/v1/translate.proto

```diff
@@ -49,7 +49,7 @@ message UploadTranslationFileRequest {
   bytes data = 2;
   Schema schema = 3;
   string service_id = 4;
-  bool original = 5;
+  optional bool original = 5;
```

Comment:
Hmm, did not know that. 

The last time I checked all proto fields were implicitly optional, but by making it explicitly optional, the field becomes a pointer in the generated file.

Nevertheless, in other languages e.g. cpp, the generated code contains `HasField` method, which can check if the field is set or not, but here we only have one getter, which returns false if not set, and the field value if it is set, so that means there is no way, to distinguish between zero value (false), and set value (false), when we are using getter...

The solution that Jānis used, is to get field omitting getter, but that beats the purpose of getter which is to ensure null safety.

Summarizing all that, got one thing to say: interesting :)